server/projects/projects.go

```diff
@@ -107,3 +107,21 @@ func GetProjectFromAPIKey(ctx context.Context, be *backend.Backend, apiKey strin
 
 	return info.ToProject(), nil
 }
+
+// GetProjectFromSecretKey returns a project from a secret key.
+func GetProjectFromSecretKey(ctx context.Context, be *backend.Backend, secretKey string) (*types.Project, error) {
+	if secretKey == "" {
+		info, err := be.DB.FindProjectInfoByID(ctx, database.DefaultProjectID)
+		if err != nil {
+			return nil, err
+		}
+		return info.ToProject(), nil
+	}
```

Comment:
I think this logic returns the default project if we don't pass a secretKey. Why is this needed?

Reply:
I was also curious about the necessity of this logic, and since `GetProjectFromAPIKey` was returning the default project in the same way, I thought there must be a reason, so I did the same.
I don't particularly think of any side effects that occur when the default project is not returned.

Reply:
To get started with Yorkie, users can easily install it using a package manager like Homebrew and they can run Yokie in standalone mode. We allows users to test the functionality without the need to create a user or project explicitly to get the apiKey.

But I don't think this feature(return default project without apiKey) needs for secretKey.

Reply:
https://github.com/yorkie-team/yorkie/pull/755/commits/b3a93275abf0d17c6ad65ff3f6f7bee7e5b93d7c